File: Mobile/objects/viewmodels/CustomerOrganizations/OwnedCustomerOrganizationViewModel.ts
import { BasicOrganizationViewModel } from "../Organizations/BasicOrganizationViewModel";
import { CustomerOrganizationsViewModel } from "../Organizations/Mobile/CustomerOrganizationsViewModel";
import { CustomerUserViewModel } from "../Users/customers/CustomerUserViewModel";

export interface OwnedCustomerOrganizationViewModel {
    id?: number;
    customerId?: string;
    customer?: CustomerUserViewModel;
    organizations?: CustomerOrganizationsViewModel[];
}

File: BE/Boek.Infrastructure/ViewModels/CustomerOrganizations/OwnedCustomerOrganizationViewModel.cs
using Boek.Infrastructure.Attributes;
using Boek.Infrastructure.ViewModels.Organizations;
using Boek.Infrastructure.ViewModels.Users.Customers;

namespace Boek.Infrastructure.ViewModels.CustomerOrganizations
{
    public class OwnedCustomerOrganizationViewModel
    {
        [Int]
        public int? Id { get; set; }

        [Guid]
        public Guid? CustomerId { get; set; }

        public CustomerUserViewModel Customer { get; set; }

        public List<CustomerOrganizationsViewModel> Organizations { get; set; }
    }
}
